tox.ini

```diff
@@ -18,6 +18,10 @@ tox_pip_extensions_ext_venv_update = true
 tox_pyenv_fallback = false
 
 [testenv]
+envdir =
+    {tests,coverage}: {toxworkdir}/monotest
+    {.tox,dev,lint,format,checkformatting,docstrings,checkdocstrings,build,pip-compile,docker-compose,update-pdfjs}: {toxworkdir}/{envname}
```

Comment:
If you don't have this comment then you have set the default to `None` which means `.`, which causes tox to fais complaining it will overwrite your project directory.

I wonder if you can just put `{toxworkdir}/{envname}` by itself without specifying the test env to re-assert the default?

Reply:
I think we should just avoid using `envdir` anyway, the alternative is both simpler and faster (see comment above)